src/main/java/digital/slovensko/autogram/core/SigningParameters.java

```diff
@@ -203,4 +212,33 @@ public int getVisualizationWidth() {
         return (visualizationWidth > 0) ? visualizationWidth : 640;
     }
 
+    public String getTransformationOutputMimeTypeString() throws TransformationParsingErrorExeption {
```

Comment:
Toto vyzera ako getter ale v skutocnosti to je dost "draha" operacia, nevieme to nastavit "zhora".